src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import task.*;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static final ArrayList<Task> userTasks = new ArrayList<>();
+    private static final String line = "____________________________________________________________";
+    private static int taskCount = 0;
+
+    public static void printGreetMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(line);
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println(line);
+    }
+
+    public static void printByeMessage() {
+        System.out.println(line);
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println(line);
+    }
+
+    public static void addTask(Task t) {
+        userTasks.add(t);
+        taskCount += 1;
+    }
+
+    public static void printMarkDone(int taskId) {
+        System.out.println(line);
+        System.out.println("\tNice! I've marked this task as done: \n\t" + userTasks.get(taskId).toString());
+        System.out.println(line);
+    }
+
+    public static void printHelpMessage() {
+        System.out.println(line);
+        System.out.println("The command you entered is invalid");
+        System.out.println("Enter one of these commands");
+        System.out.println("A. ADD A TASK");
+        System.out.println("\t1. Todo - Task with only the description\n\ttodo <task description>");
+        System.out.println("\t2. Deadline - Task with a set deadline \n\tdeadline <task description> /by <date of completion>");
+        System.out.println("\t3. Event - Task with a set date\n\tevent <task description> /at <date of event>");
+        System.out.println("B. LIST THE TASK");
+        System.out.println("\t list");
+        System.out.println("C. MARK THE TASK AS DONE");
+        System.out.println("\t done <task number>");
+        System.out.println("D. DELETE THE TASK");
+        System.out.println("\t delete <task number>");
+        System.out.println("E. END THE PROGRAM");
+        System.out.println("\t bye");
+        System.out.println(line);
+    }
+
+    private static void printNegativeTaskNumber() {
```

Comment:
Method name is a bit ambiguous, I see that this is more like an error message? So maybe printErrorMessageNegativeTaskNumber is a more accurate method description